app/assets/stylesheets/layouts/_half_width.scss

```diff
@@ -1,5 +1,4 @@
 .l-half-width {
-  float: left;
   padding-right: $gutter-width/2;
   width: percentage(1/2);
```

Comment:
Though, actually, I've already made changes to this in another branch I'll be PRing after this. Mind if I address it there?